src/simulation/dynamics/prescribedMotion/prescribedMotionStateEffector.cpp

```diff
@@ -48,12 +48,18 @@ PrescribedMotionStateEffector::PrescribedMotionStateEffector()
     this->omegaPrime_MB_B.setZero();
     this->sigma_MB.setIdentity();
 
+    this->effectorID++;
```

Comment:
Let's change this to `PrescribedMotionStateEffector::effectorID++;` . It's much clearer to refer to a static class variable like that. The this while compiling, incorrectly implies that the variable is unique to this instance of the class, which it is not.

Reply:
Done